template updates

fmichonneau opened the pull request:
This PR updates the HTML to be compatible with the new consolidated version of the CSS file and with the newest version of bootstrap.

It should only be merged once the CSS repository is created and its content merged with the workshop-template repository.

gvwilson committed:
Initializing template for Software Carpentry bootcamp home pages.

gvwilson committed:
Initializing new repository for Software Carpentry web site.

gvwilson committed:
Adding more questions to FAQ based on suggestions from current instructors.

pipitone committed:
Full-width page layout, and logo in navbar

pipitone committed:
Lightened font and search background for readability

gvwilson committed:
Merge pull request #2 from pipitone/full-width

Full-width page layout, and logo in navbar

Commit:
Solve issue #44

The 'box' class overflow the window width for some screen size.
This patch fix it using CSS media query.

Commit:
Fix for issue #44 with large font size

When using large font it will overflow the width so I add
max-width in the css.

synesthesiam committed:
First draft of skimage novice blog post

abbycabs committed:
Beginning redesign
* very much a work in progress
* experimenting with header/banner/home page

abbycabs committed:
updating gh-pages

abbycabs committed:
layout changes

abbycabs committed:
Add hammer, updates templates for new layout
* footer still needs work
* not sure if I got all the pages

abbycabs committed:
Redesign work
* add hammer to header
* add twitter to home page
* adjust styles
* adjust branding on all pages

gvwilson committed:
Reworking the home page

Commit:
Changes base on @gvwilson comments

* Avoid duplicate lines in HTML pages with layout and CSS
* Avoid import inside functions
* Warning user that the badge already in version 1.0.0
* Fix typo: backing => baking
* Avoid dict.key()
* Add words about Anaconda

abbycabs committed:
redesign updates
* simplifying design
* use header more on each page

abbycabs committed:
Merge remote-tracking branch 'sc/master' into redesign

* sc/master: (27 commits)
  Adding link to Indiana U. bootcamp
  Blog post about Andromeda Yelton coming to teach librarians
  Blog post about instructor training
  Instructor stats post
  Added links for Wisc, ESAC
  Change `requirements.txt`
  Added Dundee bootcamp (too late, it started today :( )
  Making the badges home page a bit more dramatic
  Changing wording of badges index page
  Adding Joshua Herr
  Blog post about GitHub/figshare project
  Add core and helper
  Remove old CSS from `badges/index.html`
  Update all badges to version 1.0.0
  Changes base on @gvwilson comments
  Blog post about pictures
  Rationalizing blog post labels
  Blog post about release v2013.11
  Update the creating of badges
  Import badgebakery in badge-update.py
  ...

abbycabs committed:
Redesign work - update header
* cleaner design

abbycabs committed:
Footer work

gvwilson committed:
Simplifying layout of bootcamps home page

abbycabs committed:
Footer/home page work

gvwilson committed:
Removing trailing whitespace

abbycabs committed:
add media queries - small screen support

abbycabs committed:
allow smaller screens

abbycabs committed:
search bar - change black line to blue
* issue #263

abbycabs committed:
remove text-shadow in header
* issue #264

abbycabs committed:
remove text-shadow in header again
* issue #264

abbycabs committed:
Brighter link colour
* #278

abbycabs committed:
Trying to keep all clickable item blue
* helps with learning a new ui
* 3 sub-headings on home page are no longer links

abbycabs committed:
Keep headers the darker blue
* related to #278

abbycabs committed:
override blockquote font size change
* issue #277

synesthesiam committed:
Fixed conflicts

gvwilson committed:
Changing default mailing address and putting contact email in footer

gvwilson committed:
Merge branch 'master' of github.com:synesthesiam/site into synesthesiam-master

gvwilson committed:
Editing out the notebook-specific stuff in Mike Hansen's post

abbycabs committed:
Navbar work
* mostly the collapsed the bar, make less awkward
* some link colour changes

abbycabs committed:
navbar & links
* input from @r-gaia-cs
* links on hover: darker & underline
* move search bar to bottom on collapse

abbycabs committed:
Change header case for @r-gaia-cs

gvwilson committed:
Preparing material for Version 5 novice release

gvwilson committed:
1.  Moving general information about bootcamps and setup instruction into `index.html`
    (instead of using %include directives).
2.  Deleting unused include files containing general bootcamp information.
3.  Leaving the setup instruction snippets in place for now.
4.  Modifying CSS to format things more cleanly now that they're inlined.

gvwilson committed:
Inlining contents of generic bootcamp index page

Commit:
CSS arrow to mark input and output

**Note**: This is a prototype.

Instead of "In" and "Out" use arrow to mark the input and output
of IPython lessons.

To construct the arrows we only use CSS.

abbycabs committed:
Cleaning up css
* remove unused classes copied from google form
* moved styles to swc.css
* #361

gvwilson committed:
*   Getting rid of old cheat sheets
*   Indenting error output cells
*   Removing explicit 'level' keys from Markdown files
*   Storing the generated files so that people who don't use those tools won't have to regenerate them.
*   Modifying .gitignore to reflect this.
*   Updating the Makefile to run Jekyll exactly once.
*   Command to install on the server
*   No longer worrying about making a page of images
*   Better (more guessable) name for the target that builds the website
*   Fixing image paths
*   Fixing up glossary entries

gvwilson committed:
Indenting error output cells

gvwilson committed:
Merge branch 'sticking-with-jekyll' of github.com:gvwilson/bc into sticking-with-jekyll

gvwilson committed:
Getting the pre-formatted code's color right

Commit:
Add CSS media query to improve logo

gvwilson committed:
Cleaning up novice extras

Commit:
Merge pull request #385 from r-gaia-cs/css-mobile

Add CSS media query to improve logo position

Commit:
Add GitHub Ribbon to our lessons

Backport of 9a19e91 from swcarpentry/site.

All credits to Abigail Cabunoc <abigail.cabunoc@oicr.on.ca>
for the awesome work with CSS.

Commit:
Fix btn-nav position due GitHub Ribbon

gvwilson committed:
Converting to individual challenge div's in the novice shell lessons

abbycabs committed:
Change to a css github ribbon
* responsive for smaller screens
* move menu over more - click will always open menu
* #462

abbycabs committed:
Cross browser - stop text wrap in ff
* #462

Commit:
Improve CSS for GitHub ribbon

- For small screen, preserve margin.
- Use css-beautify

gvwilson committed:
Novice shell lesson is now written in terms of Nelle instead of Vlad.

Commit:
Merge pull request #447 from r-gaia-cs/github

Add GitHub Ribbon to our lessons

Commit:
Update template for book to rely on CSS

Use h2 tag for "subtitle" will break any tool used to convert HTML
for another format. This commit replace h2 with span and rely on
CSS for have the same look.

Commit:
Update IPython Notebook template

- Improve to handle err class
- Improve pandoc support

Commit:
Add CSS for printing

The CSS was add at our main CSS and remove from printing:

- Logo at header
- GitHub ribbon
- Footnote

Close #561.

Commit:
Shrink the font size for print version

Commit:
Merge pull request #535 from gvwilson/rewriting-shell-lesson-using-nelle

Rewriting novice shell lesson so that everything refers to Nelle instead of Vlad.

Commit:
Replace bootcamp with workshop

~~~
$ for file in $(grep -r -l 'bootcamp' .); do sed -i 's/bootcamp/workshop/g' $file; done
$ for file in $(grep -r -l 'boot camp' .); do sed -i 's/boot camp/workshop/g' $file; done
~~~

and others minors changes.

Close #240.

Commit:
Changing wording in everything except blog posts

Commit:
Move files into css

Commit:
Clean CSS file

Commit:
Blockquote margins

Commit:
Remove border bottom from title

Commit:
Set up code color

Commit:
Improve callout box

and "learning objectives" box.

abbycabs committed:
Instructor training registration form
* add Country drop down select
* add Gender

gvwilson committed:
Reformatting instructor bios

gvwilson committed:
Cleaning up nav bar

katieannemills committed:
added some challenge CSS

Commit:
Small improvements to CSS

-   Remove extra space before title of boxes
-   Remove background color of boxes when printing
-   Add "Input", "Output" and "Error" tips for code blocks
    when printing
-   Avoid duplicate lines

jpallen committed:
Separate out some specific workshop classes.

The workshop template is mostly plain bootstrap, so getting it work
alongside the site CSS seems mostly straightforward. We only need to
add in a .workshop class to the body, and a .card class to the
.container in the workshop. Everything else is bootstrap content.

This commit also removes the banner.html include since this did not
seem to be removed in the main site, but was referenced in the CSS.
Removing it makes it clear the .banner class is only needed in the
workshop, not here.

gvwilson committed:
Merging @r-gaia-cs improvements

jpallen committed:
Use unified CSS from site.

As per swcarpentry/lesson-template#140, but only the core CSS
changes.

fmichonneau committed:
modifications for new CSS + bootstrap 3.3.2

fmichonneau committed:
Merge branch 'gh-pages' of https://github.com/swcarpentry/workshop-template into new-css

Conflicts:
	_includes/header.html
	_includes/javascript.html
	_layouts/page.html
	_layouts/workshop.html

jpallen commented on _includes/banner.html:
```diff
@@ -1,4 +1,3 @@
-<span id="github-ribbon"><a href="https://github.com/swcarpentry/bc">Fork me on GitHub</a></span>
```
Was this removed intentionally? Should that be a different pull request if you think the ribbon shouldn't be there?

gvwilson replied:
I saw @fmichonneau's note and added #148 as a fill-in.

fmichonneau replied:
there are 2 issues:
- from what I gathered, I think the github ribbon shouldn't be included inside the banner as the new bootstrap default messes up the positioning of the ribbon (see swcarpentry/styles#4) and therefore I moved it to workshop.html for now (side effect it won't show up on other pages, but I'll fix that once I figure out the CSS issue)
- the URL is not appropriate anymore because of the reorganization of the repositories, correctly addressed by #148

jpallen commented on _layouts/page.html:
```diff
@@ -11,14 +11,14 @@
     {% include header.html %}
   </head>
   <body class="workshop">
-    <div class="container card">
```
In the new CSS, .card denotes a white background, and I think removing it will break the style. Was it removed intentionally?

fmichonneau replied:
I don't see any issue with the formating without the card class.

fmichonneau committed:
upgrade to bootstrap 3.3.2

fmichonneau committed:
consolidate CSS files

fmichonneau committed:
fix path for bootstrap.js

fmichonneau committed:
use correct swc.css file

Commit:
Move files (from site) into css

Commit:
Merge branch 'site'

Commit:
Merge pull request #5 from swcarpentry/join-site-lesson

Join CSS from site and lessons

fmichonneau committed:
Merge github.com:swcarpentry/styles into upgrade-bootstrap

Conflicts:
	css/swc-bootstrap.css
	css/swc.css

Commit:
Merge pull request #3 from fmichonneau/upgrade-bootstrap

Upgrade bootstrap

fmichonneau committed:
Merge branch 'gh-pages' of https://github.com/swcarpentry/workshop-template into new-css

Conflicts:
	_includes/banner.html

fmichonneau committed:
Add 'css/' from commit '101aa9243c4ae16302c366796947d08716f5fd7c'

git-subtree-dir: css
git-subtree-mainline: 174a5ea3d64616bee1c39bf284e97574f5052c38
git-subtree-split: 101aa9243c4ae16302c366796947d08716f5fd7c

fmichonneau committed:
add card class to container

fmichonneau committed:
remove css/ subtree
